schemas/meta/schema.json

```
@@ -0,0 +1,47 @@
+{
+    "definitions": {
+        "field": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "description": "The type of field",
+                    "type": "string",
+                    "enum": [
+                        "date",
+                        "text",
+                        "html"
+                    ]
+                }
```

I think we discussed to add a required and a description field or did you want to leave this for later?
```suggestion
                "type": {
                    "description": "The type of field",
                    "type": "string",
                    "enum": [
                        "date",
                        "text",
                        "html"
                    ]
                },
                "required": {
                    "description": "Whether it's required",
                    "type": "boolean",
                    "default": false
                },
                "description": {
                    "description": "What is this field",
                    "type": "string"
                }
```

---

Required and description fields have been added.